snapcraft.yaml

```diff
@@ -0,0 +1,17 @@
+name: scancode-toolkit # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '1.0'  # just for humans, typically '1.2+git' or '1.3.2'
+summary: scancode-toolkit # 79 char long summary
+description: |
+  :mag_right: ScanCode detects licenses, copyrights, package manifests & dependencies and more by scanning code ... to discover and inventory open source and third-party packages used in your code.
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: devmode # use 'strict' once you have the right plugs and slots
+
+parts:
+  scancode-toolkit:
+    # See 'snapcraft plugins'
+    plugin: python
+    python-version: python2
+    source: .
+    stage-packages: 
+       - python-six
```

Comment:
I think this is not related to this PR

Reply:
yes, you are right

Reply:
i am facing some issues, what should be done to resolve this?

Reply:
What kind of issues ?

Reply:
should i close this PR and create a new one?

Reply:
Yeah that can also be done